docs/sources/reference/components/beyla.ebpf.md

```diff
@@ -135,6 +138,23 @@ It is used to populate the `service.name` OTEL property and/or the `service_name
 If the executable matches only one of the ports in the list, it is considered to match the selection criteria.
 `exe_path` accepts a regular expression to be matched against the full executable command line, including the directory where the executable resides on the file system.
 
+### kubernetes services block
+
+This block allows you to filter the services to instrument based on their Kubernetes metadata. If you specify other selectors in the same services entry,
+the instrumented processes need to match all the selector properties.
+
+Name         | Type     | Description                                                                     | Default | Required
+-------------|----------|---------------------------------------------------------------------------------|---------|---------
+`namespace` | `string` | Regular expression of Kubernetes Namespaces to match.                          |         | no
+`pod_name`  | `string` | Regular expression of Kubernetes Pods to match.                           |         | no
+`deployment_name` | `string` | Regular expression of Kubernetes Deployments to match.                 |         | no
+`statefulset_name` | `string` | Regular expression of Kubernetes StatefulSets to match.                 |         | no
+`replicaset_name` | `string` | Regular expression of Kubernetes ReplicaSets to match.                 |         | no
+`daemonset_name` | `string` | Regular expression of Kubernetes DaemonSets to match.                 |         | no
+`owner_name` | `string` | Regular expression of Kubernetes owners of running Pods to match.                     |         | no
+`pod_labels` | `map(string)` | Key-value pairs of labels with keys matching Kubernetes Pods with the provided value as regular expression.             |         | no
```

Comment:
To align with other documentation pages these should have defaults filled out to the empty value; strings should be documented with a default of `""`, and `map(string)`s should be documented with a default of `{}`.